cloudinit/distros/opensuse.py

```diff
@@ -66,15 +74,72 @@ def install_packages(self, pkglist):
         )
 
     def package_command(self, command, args=None, pkgs=None):
+        pckg_op = command
         if pkgs is None:
             pkgs = []
 
+        if self.update_method is None:
```

Comment:
this whole if could be a helper function to make this function easier to read

Reply:
I agree. Something like a `set_update_method` would make it easier to follow.

Reply:
Done